datasets/ccaligned_multilingual/ccaligned_multilingual.py

```diff
@@ -0,0 +1,213 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CCAligned Multilingual Translation Dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{elkishky_ccaligned_2020,
+ author = {El-Kishky, Ahmed and Chaudhary, Vishrav and Guzm{\'a}n, Francisco and Koehn, Philipp},
+ booktitle = {Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP 2020)},
+ month = {November},
+ title = {{CCAligned}: A Massive Collection of Cross-lingual Web-Document Pairs},
+ year = {2020}
+ address = "Online",
+ publisher = "Association for Computational Linguistics",
+ url = "https://www.aclweb.org/anthology/2020.emnlp-main.480",
+ doi = "10.18653/v1/2020.emnlp-main.480",
+ pages = "5960--5969"
+}
+"""
+
+_DESCRIPTION = """\
+CCAligned consists of parallel or comparable web-document pairs in 137 languages aligned with English. These web-document pairs were constructed by performing language identification on raw web-documents, and ensuring corresponding language codes were corresponding in the URLs of web documents. This pattern matching approach yielded more than 100 million aligned documents paired with English. Recognizing that each English document was often aligned to mulitple documents in different target language, we can join on English documents to obtain aligned documents that directly pair two non-English documents (e.g., Arabic-French).
+"""
+
+_HOMEPAGE = "http://www.statmt.org/cc-aligned/"
+
+
+_LICENSE = ""  # Unknown
+
+
+_URLs = {
+    "documents": "http://www.statmt.org/cc-aligned/",
+    "sentences": "http://www.statmt.org/cc-aligned/sentence-aligned/",
+}
+
+reverse_mapped_sentences = [
+    "af_ZA",
+    "ak_GH",
+    "am_ET",
+    "ar_AR",
+    "as_IN",
+    "ay_BO",
+    "az_AZ",
+    "az_IR",
+    "be_BY",
+    "bg_BG",
+    "bm_ML",
+    "bn_IN",
+    "br_FR",
+    "bs_BA",
+    "ca_ES",
+    "cb_IQ",
+    "cs_CZ",
+    "cx_PH",
+    "cy_GB",
+    "da_DK",
+    "de_DE",
+    "el_GR",
+]  # Some languages have the reverse source languages in the URLs.
+
+
+class CCAlignedMultilingualConfig(datasets.BuilderConfig):
+    def __init__(self, *args, type=None, language_code=None, **kwargs):
+        super().__init__(
+            *args,
+            name=f"{type}-{language_code}",
+            **kwargs,
+        )
+        self.type = type
+        self.language_code = language_code
+
+
+class CCAlignedMultilingual(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class CcalignedMultilingual(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script `ccaligned_multilingual`